store/src/java/com/zimbra/cs/account/ZAttrAccount.java

```diff
@@ -34977,7 +35049,7 @@ public void setPasswordBlockCommonEnabled(boolean zimbraPasswordBlockCommonEnabl
      * @param attrs existing map to populate, or null to create a new map
      * @return populated map to pass into Provisioning.modifyAttrs
      *
-     * @since ZCS 8.9.0
+     * @since ZCS 9.1.0
```

Comment:
why are we changing since values of existing attributes?

Reply:
This file is system generated. and i didn't change anything in this attribute. Don't know how can we remove this changes.

Reply:
rebase or check zimbra-attrs.xml